### ISSUE TITLE
ssh never becomes available

### ISSUE THREAD

**Issue by EugenMayer** [janeczku/docker-machine-vultr#35]
I am using the vulture machine driver, deploying using the rancher ui.

Using region 9, default endpoint, my ssh personal ssh key and ros v1.1.1, rancher-server 1.6.13

The deployment starts, the server gets deployed on vultr, i can connect using SSH personally, but the rancher server still "waits for available ssh connection". Is the whole point that the rancher-server instance is actually trying to ssh into that sever using its own SSH key ( i guess ) so i would need to reference that in the ssh key id section?

thank you for this great driver!

**Comment by EugenMayer** [comment]
Just in case, that helps, upgraded to your machine driver version 1.4, but still getting the same issue 

![image](https://user-images.githubusercontent.com/136934/34320799-99a4560c-e801-11e7-9992-8c00908d223b.png)

while ssh already works

```
ssh rancher@45.77.140.42
[rancher@w2 ~]$
```

**Comment by EugenMayer** [comment]
Could it be related to the fact that i do not use a reservedIpID at all so rancher does not know the IP of the host being setup by vultr?

My data is:
region: 9 (Frankfurt)
ssh-key: <mysshkeyidinvultr>
rancherosversion: 1.1.1
apikey: <myipkey>

anything else is blank / default

**Comment by EugenMayer** [comment]
Last information, adding the host using the vultr driver, then removing the host in the rancher GUI, then ssh'ing into the host and using "custom host" to deploy a rancher agent will properly register the host with the rancher server. So there is basically all setup, just SSH seems to be broken.

What i ask myself though is, how is the machine-driver actually SSH-ing without having my SSH private key - i guess the ssh connection is used to deploy the rancher agent? Is a vultr api used for that? Kind of makes no sense here

**Comment by byjg** [comment]
I created the machine outside the Rancher GUI using the vultr machine driver 1.4.0 with the command below:

```
docker-machine create \
     --driver vultr --vultr-api-key=00000000 \
     --vultr-private-networking=true --vultr-ssh-key-id 0000000000 \
     my-machine
```
The machine was created and the SSH is OK. I can connect with the user "rancher" in the machine. 

In the Rancher GUI interface I leave all parameters blank except the parameters I setup above. 

So the list of problems I notice / have:

- Rancher GUI is passing some king of parameter to Vultr Docker Machine Driver that the Vultr Driver does not recognize and it is causing the error;
- The default user is "rancher" but in the Rancher GUI interface is "root" (This is only a note)
- The interface eth1 on the machine created is UP but does not have any IP. I had to setup manually.

Version of the software I used:
- Vultr Docker Machine Driver: 1.4.0 
- Ranche Server: v1.6.13
